csm/conf/setup.py

```diff
@@ -68,6 +69,8 @@ def _set_deployment_mode(self):
         if self._setup_info[const.NODE_TYPE] == const.VM:
             Log.info("Running Csm Setup for VM Environment Mode.")
             self._is_env_vm = True
+        if Conf.get(const.CONSUMER_INDEX, const.DEPLOYMENT_MODE) == "DEV":
```

Comment:
Deployment mode was changed to lowercase in `templates/csm_setup_conf_template.json` but not here. I think we need to do as with the rest of the code where we use the constant defined on the all-encompassing `const` module.

Also, it is good practice to run case-insensitive checks for strings used as state enumerations.

Reply:
Done the changes. 
This was Skipped for some reason fixed by using a single key from const file.